drivers/lora/sx126x.c

```diff
@@ -296,7 +281,7 @@ void SX126xSetOperatingMode(RadioOperatingModes_t mode)
 
 uint32_t SX126xGetBoardTcxoWakeupTime(void)
 {
-	return TCXO_POWER_STARTUP_DELAY_MS;
+	return DT_INST_PROP_OR(0, tcxo_power_startup_delay_ms, 0);
```

Comment:
Defines are not only used to avoid code duplication but also to provide code simplicity. They provide a meaningful flow to the code and I prefer to keep them.

Reply:
Can you please explain how splitting this logic across 2 locations in the file makes the code simpler?